### PR TITLE
Feature adding hover on indicator

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**

### CODE REVIEW

File: src/store/ChartAdapterStore.ts [hunk 1]
```diff
@@ -58,15 +163,9 @@ export default class ChartAdapterStore {
...
-                // dxLocal and dyLocal are the local position value correponding to the bottom indicator/main chart
-                const epoch = this.flutterChart?.crosshair.getEpochFromX(dxLocal) || 0;
-                const quote = (this.flutterChart?.crosshair.getQuoteFromY(dyLocal) || 0).toFixed(
-                    this.mainStore.crosshair.decimalPlaces
-                );
-
-                this.mainStore.crosshair.onMouseMove(dx, dy, epoch, quote);
+            onCrosshairHover: (dx, dy, dxLocal, dyLocal, bottomIndicatorIndex) => {
+                const debounced = this.debounce(this.crossHover, 100);
+                debounced(dx, dy, dxLocal, dyLocal, bottomIndicatorIndex);
```
Comment (balakrishna-deriv) [inline src/store/ChartAdapterStore.ts:168]: can you use debounce from 'lodash.debounce'? We already have that in our package.json

Comment (balakrishna-deriv) [reply src/store/ChartAdapterStore.ts:168]: Let's not do debounce for tooltip move. Tooltip won't take that much computing and it has to be smooth.
Can we do debounce only for indicator hover check?

File: package-lock.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "@deriv/deriv-charts",
-    "version": "1.3.2",
-    "lockfileVersion": 3,
+    "version": "1.3.1",
+    "lockfileVersion": 2,
```
Comment (balakrishna-deriv) [inline package-lock.json:4]: why are we changing this here?

Comment (ahmadtaimoor-deriv) [reply package-lock.json]: fixed now , it got changed during merge conflict fixes , Thanks

File: src/store/ChartAdapterStore.ts (hunk 1 above)
Comment (ahmadtaimoor-deriv) [reply src/store/ChartAdapterStore.ts]: lodesh.debounce is used now . Debounce for hover is only done now , thanks

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**